app/main/services/github_service.py

```diff
@@ -14,8 +14,10 @@ def __init__(self, org_token: str, repository: str) -> None:
 
     def submit_issue(self, form_data: dict) -> None:
         title = f"[DNS] {form_data['domain_name']}"
+        print(form_data)
```

Comment:
Could we use logging here instead of print? 🥺 Logging shows up a lot better in the console, maybe set to debug as well if we're quite confident that the form passes through okay 

```suggestion
        logging.debug(form_data)
```

Reply:
sorry that's debug will remove